editoast/src/client/stdcm_search_env_commands.rs

```diff
@@ -394,6 +423,7 @@ mod tests {
             work_schedule_group_id: Some(work_schedule_group.id),
             search_window_begin: None,
             search_window_end: None,
+            active_perimeter_geojson_path: None,
```

Comment:
Do you think you can test when the `active_perimeter_geojson_path` field is `Some` ? I don't know how difficult it is 😕

Reply:
Done in one test. I let the other one with None to test it either